lib/fluent/plugin/out_logdna.rb

```diff
@@ -10,7 +10,7 @@ class LogDNAOutput < Output
     config_param :ip, :string, default: nil
     config_param :app, :string, default: nil
     config_param :file, :string, default: nil
-    config_param :ingester_domain, :string, default: 'https://logs.logdna.com'
+    config_param :ingester_host, :string, default: 'https://logs.logdna.com'
```

Comment:
nit: this is more of a URL than a host

Reply:
yeah… well… meh, this is just my silly semantics. I don't care much either way. Their other agent uses LOGDNA_LOGHOST: https://github.com/logdna/logdna-agent/blob/a2e23f2203834728721765077ae160e89510fc2c/lib/config.js#L10

It's not the full url, the path gets added later. meh